app/models/client.rb

```diff
@@ -36,4 +36,9 @@ def self.create_from_intake(intake)
       vita_partner: intake.vita_partner,
     )
   end
+
+  # Returns the phone number formatted for user display, e.g.: "(510) 555-1234"
+  def formatted_phone_number
+    Phonelib.parse(phone_number).local_number
+  end
```

Comment:
We have a bunch of these formatted_phone_number methods spread around the code base. It might make sense to create a view helper for this, thereby making a method that can be used in any template for any model.

```ruby
# in app/helpers
module PhoneNumberHelper
  def local_phone_format(phone_number)
    Phonelib.parse(phone_number).local_number
  end
end
```

```html.erb
<%# in any template %>
<%= local_phone_format(client.phone_number) %>
```

Reply:
will do this as part of the next one!